File: practice_exam/03.c
/*
	Program that:
		Dado un número, muestre la suma de sus dígitos
*/

#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main (void) {
	int number = 0;
	int sum = 0;

	scanf("%d", &number);

	while (number > 0) {
		sum += number % 10;
		number /= 10;
	}

	printf("%d", sum);

	return 0;
}

File: practice_exam/03.java
/* 
	Program that:
		Dado un número, muestre la suma de sus dígitos.
*/

import java.util.Scanner;

public class Main {
	public static void main (String[] args) {
		Scanner userInput = new Scanner(System.in);
		int sum = 0;
		int number = 0;

		number = userInput.nextInt();

		while (number > 0) {
			sum += number % 10;	
			number /= 10;
		}

		System.out.printf("%d", sum);

		userInput.close();
	}
}
